extensions/adobe/experience/content-telemetry/webpage.schema.json

```diff
@@ -0,0 +1,111 @@
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/content-telemetry/webpage",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Telemetry Web Page",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Web page details and containing web elements",
+  "definitions": {
+    "webpage": {
+      "properties": {
+        "repo:id": {
+          "title": "Page Id",
+          "type": "string",
+          "meta:immutable": true,
+          "meta:usereditable": false,
+          "description": "The unique ID of the web page."
+        },
+        "repo:name": {
+          "meta:usereditable": false,
+          "title": "Page name",
+          "type": "string"
+        },
+        "dc:title": {
+          "title": "Title",
+          "type": "string",
+          "description": "The title of the web page"
+        },
+        "repo:lastModifiedDate": {
+          "type": "string",
+          "format": "date-time",
+          "meta:usereditable": false,
+          "examples": ["2004-10-23T12:00:00-06:00"],
+          "description": "The server date and time when the resource was last modified in the repository, such as when a new version of an asset is uploaded or a directory's child resource is added or removed. The date time property should conform to ISO 8601 standard. An example form is \"2004-10-23T12:00:00-06:00\"."
+        },
+        "xdm:pageType": {
+          "title": "Type",
+          "type": "string",
+          "description": "The type of the component."
+        },
+        "xdm:templatePath": {
+          "title": "Page Template path",
+          "type": "string",
+          "description": "The path to the page's template"
+        },
+        "xdm:language": {
+          "type": "string",
+          "pattern": "^(((([A-Za-z]{2,3}(-([A-Za-z]{3}(-[A-Za-z]{3}){0,2}))?)|[A-Za-z]{4}|[A-Za-z]{5,8})(-([A-Za-z]{4}))?(-([A-Za-z]{2}|[0-9]{3}))?(-([A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-([0-9A-WY-Za-wy-z](-[A-Za-z0-9]{2,8})+))*(-(x(-[A-Za-z0-9]{1,8})+))?)|(x(-[A-Za-z0-9]{1,8})+)|((en-GB-oed|i-ami|i-bnn|i-default|i-enochian|i-hak|i-klingon|i-lux|i-mingo|i-navajo|i-pwn|i-tao|i-tay|i-tsu|sgn-BE-FR|sgn-BE-NL|sgn-CH-DE)|(art-lojban|cel-gaulish|no-bok|no-nyn|zh-guoyu|zh-hakka|zh-min|zh-min-nan|zh-xiang)))$",
+          "description": "Specifies the language of this web element. the language property should conform to BPC 47, for example `en-GB`."
+        },
+        "xdm:description": {
+          "title": "Page description",
+          "type": "string",
+          "description": "Page description."
+        },
+        "xmp:keywords": {
```

Comment:
see previous comments on `xmp:keywords`
And since you are using the same thing twice, then break it out into its own schema and reference from both.